quoridorui/src/main/java/com/harush/zitoon/quoridor/ui/controller/AbstractController.java

```diff
@@ -28,8 +28,11 @@ protected void loadScreen(Stage primaryStage, String layoutFile){
 			primaryStage.setTitle("Quoridor");
 			primaryStage.getIcons().add(new Image("resources/icons/favicon.png"));
 			primaryStage.setScene(scene);
-			((AbstractController)root.getController()).postConstruct();
-			primaryStage.show();
+			primaryStage.setFullScreenExitHint("");
+            primaryStage.setFullScreen(true);
```

Comment:
Please try to maintain proper indentation. A good practice is using `CTRL` + `ALT`+ `l` after you finish writing the code.